src/open_inwoner/utils/ckeditor.py

```diff
@@ -28,15 +30,77 @@ def get_rendered_content(content):
         for element in soup.find_all(tag):
             element.attrs["class"] = class_name
             if element.name == "a" and element.attrs.get("href", "").startswith("http"):
-                icon = soup.new_tag("span")
-                icon.attrs.update(
-                    {
-                        "aria-hidden": "true",
-                        "aria-label": "Opens in new window",
-                        "class": "material-icons",
-                    }
-                )
-                icon.append("open_in_new")
-                element.append(icon)
```

Comment:
I notice the original of this function has more code then what is left after splitting it for the product version. Eg the new `get_rendered_content()` doesn't add the icon but adds a _blank target.

Reply:
Yes you are right, I did this in a generic way as we had it before [https://taiga.maykinmedia.nl/project/open-inwoner/task/1172](url). Because the icon was added only for the product view.